samples/ChatSample/ChatSample.CoreApp3/Properties/launchSettings.json

```diff
@@ -23,6 +23,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development",
         "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.Azure.SignalR"
```

Comment:
>"ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.Azure.SignalR" [](start = 8, length = 64)

You can add this configuration in below to skip the call `AddAzureSignalR()` in Startup.cs. Otherwise, it will use local SignalR. It's available in NetCoreApp3.0 only.